shiro.ini.multi-realm-example

```diff
@@ -0,0 +1,32 @@
+[main]
+securityManager.subjectDAO.sessionStorageEvaluator.sessionStorageEnabled = false
+
+; This is using DefaultLdapRealm shipped with Shiro.
+; It'll replace {0} placeholder with username supplied via SASL when connecting to Cassandra
+; and try bind with that DN and supplied password.
+
+ldapRealm = org.apache.shiro.realm.ldap.DefaultLdapRealm
+ldapRealm.userDnTemplate = cn={0},ou=users,dc=my-company,dc=com
```

Comment:
why placeholder is with zero here and without zero in `shiro.ini.example`?

Reply:
This is the config for the stock Shiro realm - should I change our realm to use `{0]` too?

Reply:
I would say yes, to make it consistent, unless there are reasons not to.

Reply:
👍